vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js

```diff
@@ -342,8 +342,6 @@ const TouchableWithoutFeedback = ((createReactClass({
       onResponderTerminate: this.touchableHandleResponderTerminate,
       onKeyUp: this._onKeyUp,
       onKeyDown: this._onKeyDown,
```

Comment:
There are more events removed in RN61 TouchableWithoutFeedBack.js, for example: onKeyUp and onKeyDown, should we remove those events too?

Reply:
I think we should, but looks like ReactNative is not doing it? https://github.com/facebook/react-native/blob/master/Libraries/Components/Touchable/TouchableWithoutFeedback.js#L75-L94

Removing it here and adding it to the override list basically means if there's no event handler set on Touchable, use the callbacks on children to handle the events, which I think makes sense for keyUp/Down events, but I'm not sure why they are not in ReactNative.

Reply:
Oh, is keyUp/Down windows only?

Reply:
I think so. Search for topKeyDown definition.

---
In reply to: [377886796](https://github.com/microsoft/react-native-windows/pull/4078#discussion_r377886796) [](ancestors = 377886796)

Reply:
yes keyUp/keyDown are windows-specific.  See [https://github.com/microsoft/react-native-windows/pull/2623](https://github.com/microsoft/react-native-windows/pull/2623).

---
In reply to: [377889221](https://github.com/microsoft/react-native-windows/pull/4078#discussion_r377889221) [](ancestors = 377889221,377886796)

Reply:
Looks like we are re-routing the events to onPress so we probably shouldn't remove them.

https://github.com/microsoft/react-native-windows/blob/master/vnext/src/Libraries/Components/Touchable/TouchableHighlight.windows.js#L453-L474

Reply:
@ddalp just checked there's no keyDown/Up events on TextInput, there's only onKeyPress. We are sending out the onPressed event in _onKeyUp callback, so I think we should keep it as is.

Also added a test to verify keyPress event on TextInput.